src/sentry/templates/sentry/bases/auth.html

```diff
@@ -1,5 +1,8 @@
 {% extends "sentry/bases/modal.html" %}
 
+{% block preload_data %}
+{% endblock %}
```

Comment:
Does this do anything?  The outer HTML is rendered.  This block gives you the opportunity to insert more within.

Reply:
It overrides the block in layout so that the include is not included (I think)